src/fun/counting/main.go.tmpl

```diff
@@ -0,0 +1,89 @@
+{{/*
+        Counting command. Count and have fun!
+        
+        Author: H1nr1 <https://github.com/H1nr1>
+*/}}
+
+{{/* Configurable Values */}}
+{{ $Admin := 123456789 }} {{/* Administrator role ID; EDIT THIS TO A ROLE ID YOU HAVE */}}
+{{/* Optional */}}
+{{ $CountTwice := true }} {{/* Allow users to count multiple times in a row; true/false */}}
+{{ $RoleID := false }} {{/* Counting role ID; set to false to disable */}}
+{{ $SecondChance := true }} {{/* Second chance if wrong; true/false */}}
+{{ $Leaderboard := true }} {{/* If you added the leaderboard CC; true/false */}}
+{{ $CorrectEmoji := "✅" }} {{/* Emoji to react with if number is correct; if custom, use format name:id */}}
+{{ $WarningEmoji := "⚠️" }} {{/* Emoji to react with if wrong number AND Second Chance is true/on; if custom, use format name:id */}}
+{{ $IncorrectEmoji := "❌" }} {{/* Emoji to react with if number is incorrect; if custom, use format name:id */}}
+{{/* End of configuration values */}}
+
+{{/* No Touchy */}}
+{{ with .ExecData }}
+	{{ if not (getMessage (toInt (dbGet 0 "Channel").Value) .) }} {{/* Check if number was deleted */}}
```

Comment:
"Channel" is a very generic db key, runs the risk of conflict with existing db entries on a user's server. This is an issue with most of your db keys.

Reply:
Removed as nil works just fine for channel value.
For other keys, I considered adding a "C" prefix, but didn't.